.travis.yml

```diff
@@ -54,6 +54,8 @@ matrix:
       env: SYMFONY=3.3.*
     - php: '7.2'
       env: SYMFONY=3.4.*
+    - php: '7.2'
+      env: SYMFONY=4.0.*
```

Comment:
This file is handled by dev-kit

Reply:
ah sorry, correct. will see if builds are green and do really add Symfony 4.0 and remove the lines then,ok?

Reply:
You will probably get an error on this build, because some dependencies might not be compatible with SF 4.0